src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1672,6 +1672,14 @@ final public static function willBeAvailable(string $package, string $class, arr
         return false;
     }
 
+    /**
+     * @internal
```

Comment:
making it `@internal` while it is used in a different package looks wrong to me, as it means FrameworkBundle cannot rely on semver guarantees of the DI component when calling that method (and so we would have to stop using the `^` operator)